samples/conditional_video_processing/demo.yml

```diff
@@ -0,0 +1,179 @@
+# module name, required
+name: ${oc.env:MODULE_NAME, 'demo'}
+
+# base module parameters
+parameters:
+  # pipeline processing frame parameters
+  frame:
+    width: 1280
+    height: 720
+    # Add paddings to the frame before processing
+    padding:
+      # Paddings are kept on the output frame
+      keep: true
+      left: 0
+      right: 0
+      # Padding size is 180px on the top of the frame
+      top: 180
+      bottom: 0
+  output_frame:
+    codec: h264
+    # Encode only frames with tag "encode"
+    when_tagged: encode
+  # PyFunc for drawing on frames
+  draw_func:
+    # specify the drawfunc's python module
+    module: samples.peoplenet_detector.overlay
```

Comment:
remove custom drawfunc, default draw func must be used configured to draw only 'peoplenet.person' objects.